### DISCUSSION TITLE
Don’t know shutdown or wake command

### DISCUSSION THREAD

**Discussion by Crox77** [AlexGustafsson/homebridge-wol#163] [Questions]
Hi, I’m in need of some help please?
I have installed the plugin and added my iMac to HomeKit and it shows the status of the Mac no problem, the problem I have is I dont know what the command lines are for on and off to add to my config.
How do I find out what they are?
Thanks
Paul

**Comment by AlexGustafsson** [comment]
I've moved your issue over to a discussion.

An example configuration from the README:

```json
{
     "accessory": "NetworkDevice",
     "name": "My Macbook",
     "mac": "<mac-address>",
     "host": "macbook.local",
     "pingCommand": "ssh macbook.local 'if [[ $(pmset -g powerstate IODisplayWrangler | tail -1 | cut -c29) -lt 4 ]]; then; exit 1; else; echo 1; fi;'",
     "pingInterval": 45,
     "wakeGraceTime": 10,
     "wakeCommand": "ssh macbook.local caffeinate -u -t 300",
     "shutdownGraceTime": 15,
     "shutdownCommand": "ssh macbook.local sudo shutdown -h now"
   },
```

Here, `macbook.local` is the domain name or the IP address of your Mac. You will have to use SSH and key-based authentication, which is out-of-scope to help you with here. For help with that, I'd recommend you to check out superuser.com or apple.stackexchange.com. There are some other issues where we've handled this sort of configuration before, such as: https://github.com/AlexGustafsson/homebridge-wol/discussions/158, https://github.com/AlexGustafsson/homebridge-wol/issues/128 and https://github.com/AlexGustafsson/homebridge-wol/discussions/154.

**Reply by fender7083** [reply]
What the command for PC. They aren't listed, only the commands for Mac OS. The only PC command I see is net rpc shutdown --ipaddress 192.168.1.15 --user fork%1965
